1.0-E2E-status.md

```diff
@@ -124,7 +124,7 @@ Legend:
 |                              | Java         | Weidong   | :running_man:            | 12/31   | [Code](https://github.com/allenjzhang/typespec-e2e-demo/tree/main/petstore/clients/java)     | `clientcore` not published to Maven                           |
 |                              | Python       | Yuchao    | :green_circle:           | 12/31   | [Code](https://github.com/allenjzhang/typespec-e2e-demo/tree/main/petstore/clients/python)   |                                                               |
 | ToDo App                     |              |           |                          |         |                                                                                              |                                                               |
-|                              | C#           | Dapeng    | :running_man:            | 1/15    | [Code](https://github.com/allenjzhang/typespec-e2e-demo/tree/main/todoApp/clients/dotnet)    | `http-client-csharp` now has a bug on multi-line descriptions |
+|                              | C#           | Dapeng    | :green_circle:           | 1/15    | [Code](https://github.com/allenjzhang/typespec-e2e-demo/tree/main/todoApp/clients/dotnet)    |  |
```

Comment:
Have we resolved all open issues/ feature asks for .NET?

Reply:
let me change it back since paging and namespace changes have not been applied